content/chat/rooms/index.textile

```diff
@@ -0,0 +1,121 @@
+---
+title: Rooms
+meta_description: "Use rooms to organize your users and chat messages."
+product: chat
+languages:
+  - javascript
+---
+
+Rooms are used to organize and logically separate your users and chat messages into 'rooms'. They are the entry object into using chat and provide access to all other chat features, such as messages, online status and typing indicators. A room can represent a 1:1 chat between an agent and a customer, a private message between two users in a chat application, a large group conversation, or the chat section of a livestream with thousands of users.
+
+You also control which features are enabled in a room when you create one. For instance, you might only want to utilize typing indicators in direct messages between users.
+
+h2(#create). Create or retrieve a room
+
+Users send messages to a room and subscribe to the room in order to receive messages. Other features, such as indicating which users are online, or which users are typing are configured as part of a room's options.
+
+A @room@ is created, or an existing one is retrieved from the @rooms@ collection using the "@rooms.get()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/Rooms.html#get method:
+
+```[javascript]
+const room = chatClient.rooms.get('basketball-stream');
+```
+
+When you create or retrieve a room using @rooms.get()@, you need to choose which additional chat features you want enabled for that room. This is configured by passing a "@RoomOptions@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/RoomOptions.html object as the second argument. In addition to setting which features are enabled, @RoomOptions@ also configures the properties of the associated features, such as the timeout period for typing indicators.
+
+```[javascript]
+const room = chatClient.rooms.get('basketball-stream', {presence, reactions, typing, occupancy});
+```
+
+You can also use the @RoomOptionsDefaults@ property for each @RoomOption@ to configure whether the default settings should be used:
+
+```[javascript]
+const room = chatClient.rooms.get('basketball-stream', {presence: RoomOptionsDefaults.presence});
+```
+
+Or configure each feature using your own values:
+
+```[javascript]
+const room = chatClient.rooms.get('basketball-stream', {typing: {timeoutMs: 5000}});
+```
+
```

Comment:
I'd offer the example which enables all features with default options for copy-pasting, but this is fine as-is too.